perfkitbenchmarker/linux_benchmarks/cloudsuite_websearch_benchmark.py

```diff
@@ -12,197 +12,129 @@
-  num_clients = FLAGS.cs_websearch_num_clients
-  config['vm_groups']['clients']['vm_count'] = num_clients
+  if FLAGS['num_vms'].present:
+    config['vm_groups']['clients']['vm_count'] = FLAGS.num_vms
   return config
 
 
-def _PrepareSolr(solr_nodes):
-  """Starts and configures SolrCloud."""
-  basic_configs_dir = posixpath.join(solr.SOLR_HOME_DIR, 'server/solr/'
-                                     'configsets/basic_configs/conf')
-  server_heap_size = FLAGS.cs_websearch_server_heap_size
-  for vm in solr_nodes:
-    vm.Install('solr')
-    vm.RemoteCommand('cd {0} && wget {1}'.format(
-        basic_configs_dir,
-        SCHEMA_URL))
-    vm.RemoteCommand('cd {0} && wget {1}'.format(
-        basic_configs_dir,
-        SOLR_CONFIG_URL))
-    if vm == solr_nodes[0]:
-      solr.StartWithZookeeper(vm, SOLR_PORT, server_heap_size)
-    else:
-      solr.Start(vm, SOLR_PORT, vm, SOLR_PORT + 1000, server_heap_size)
-  solr.CreateCollection(solr_nodes[0], 'cloudsuite_web_search',
-                        len(solr_nodes), SOLR_PORT)
-
-
-def _BuildIndex(solr_nodes):
-  """Downloads Solr index and set it up."""
-  for vm in solr_nodes:
-    vm.RemoteCommand('cd {0} && '
-                     'bin/solr stop -p {1}'.format(
-                         solr.SOLR_HOME_DIR, SOLR_PORT))
-
-  def DownloadIndex(vm):
-    solr_core_dir = posixpath.join(vm.GetScratchDir(), 'solr_cores')
-    vm.RobustRemoteCommand('cd {0} && '
-                           'wget -O - {1} | '
-                           'tar zxvf - -C {2}'.format(
-                               solr_core_dir, INDEX_URL,
-                               'cloudsuite_web_search*'))
-
-  vm_util.RunThreaded(DownloadIndex, solr_nodes, len(solr_nodes))
-  server_heap_size = FLAGS.cs_websearch_server_heap_size
-  for vm in solr_nodes:
-    if vm == solr_nodes[0]:
-      solr.StartWithZookeeper(vm, SOLR_PORT, server_heap_size, False)
-    else:
-      solr.Start(vm, SOLR_PORT, solr_nodes[0], SOLR_PORT + 1000,
-                 server_heap_size, False)
-
-
-def _PrepareClient(clients):
-  """Prepares client machine in the benchmark."""
-  client_master = clients[0]
-  for client in clients:
-    client.Install('faban')
-    client.RemoteCommand('cd {0} && '
-                         'wget -O - {1} | '
-                         'tar -xzf -'.format(
-                             faban.FABAN_HOME_DIR, SEARCH_DRIVER_URL))
-    client.RemoteCommand('cd {0}/search && '
-                         'sed -i "/faban.home/c\\faban.home={0}" '
-                         'build.properties && '
-                         'sed -i "/ant.home/c\\ant.home='
-                         '{1}" build.properties && '
-                         'sed -i "/faban.url/c\\faban.url='
-                         'http://localhost:9980/" build.properties'.format(
-                             faban.FABAN_HOME_DIR, ANT_HOME_DIR))
-  faban.Start(client_master)
-  time.sleep(20)
+def _HasDocker(vm):
```

Comment:
This function was added to linux_packages/docker.py (and named IsInstalled) as part of a separate cloudsuite benchmark pull request. Use that function instead.